doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -3617,6 +3617,7 @@ The fully expanded example above (without environment variables) looks like this
 Currently the following methods are used to detect file types:
 
 - The file type detected by the browser (or sent via API).
+- Custom code that reads the first few bytes. As explained at :ref:`s3-direct-upload-features-disabled`, this code is disabled during direct upload to S3. However, this code is active when the "redetect" API is used.
```

Comment:
I agree it's a good idea to spell this out in the guide; especially if it has been a source of confusion. 
But, somewhat of a nitpick - I'm wondering if there's a better way to say this than "this code is disabled". Something along the lines of 
"... this method of file type detection is not utilized during direct upload to S3, since by nature of direct upload Dataverse never sees the contents of the file." 
Your call. I just feel like, technically, it is more like "does not apply" or "cannot be used" than "disabled".